### ISSUE TITLE
UICircularRingValueFormatter structs have internal initializers

### ISSUE THREAD

**Issue by milbin** [luispadron/UICircularProgressRing#175]
:warning: **This issue will be closed if the required information is not filled** :warning:

_If it's a question about something that is happening within your code base, either post a small amount of code, or post an example project where I can try to assist.
Make sure to communicate back and keep issue updated or it will be closed_

## Version

- Version = 6.0.1

## Details

- Has the question been answered [here](https://github.com/luispadron/UICircularProgressRing/issues?utf8=✓&q=label%3Aquestion) before?
  - No
- Please provide as much detail as possible below
  - I cannot for the life of me, after many hours of staring at documentation figure out how to toggle the show floating point variable. I would greatly appreciate it if someone could give me a swift example of how to show floating points on the label. This is what i have been trying so far:

class CourseViewController: UIViewController {

    @IBOutlet weak var progressRing: UICircularProgressRing!
    override func viewDidLoad() {
        super.viewDidLoad()
        progressRing.font = UIFont.boldSystemFont(ofSize: 30.0)
        progressRing.value = 87.6
        progressRing.valueFormatter.showFloatingPoint = true
        progressRing.valueFormatter.decimalPlaces = 2
        // Do any additional setup after loading the view.
    }

however i get this error:"Value of type 'UICircularRingValueFormatter' has no member 'showFloatingPoint'" and "Value of type 'UICircularRingValueFormatter' has no member 'decimalPlaces'"

**Comment by luispadron** [comment]
Hi, you will need to instantiate a `UICircularProgressRingFormatter` which is a subclass of `UICircularRingValueFormatter`. The base class has no members, the former does. 

Then assign this formatter to the formatter of the ring and that should work.

**Comment by milbin** [comment]
Sorry still not sure what exactly you mean by that. Would you mind showing me with some swift code?

**Comment by luispadron** [comment]
If that’s the case, that’s definitely a bug. Give me a few minutes i’ll see what’s going on.

**Comment by luispadron** [comment]
Should be fixed in version `6.0.2`

Here is an example if it helps:

```swift
let formatter = UICircularProgressRingFormatter(showsFloatingPoint: true)
ring.valueFormatter = formatter
```

**Comment by luispadron** [comment]
Please reopen if you experience any issues still
